src/imitation/scripts/config/train_adversarial.py

```diff
@@ -140,6 +140,9 @@ def cartpole():
 def mountain_car():
     env_name = "MountainCar-v0"
     rollout_hint = "mountain_car"
+    # Original paper used negative rewards, as this is a variable-length episode
+    # environment with only negative rewards.
+    discrim_net_kwargs = {"gail": {"positive_rewards": False}}
```

Comment:
If we're just trying to match the paper results to verify the implementation then... OK. But I do want to be clear this is a horrible hack. I think how our performance in the fixed-horizon version of MountainCar matches that of other GAIL implementations is likely to be much more informative. Should we not at least have a config for those?

Reply:
Adding a config for seals:MountainCar-v0 seems good.